hoomd/md/EvaluatorPairExpandedMie.h

```diff
@@ -189,8 +189,11 @@ class EvaluatorPairExpandedMie
 
             if (energy_shift)
                 {
-                Scalar rcutninv = fast::pow(rcutsq, -n_pow / Scalar(2.0));
-                Scalar rcutminv = fast::pow(rcutsq, -m_pow / Scalar(2.0));
+                Scalar rcut = fast::sqrt(rcutsq);
+                Scalar r_cut_shifted = rcut - delta;
+
+                Scalar rcutninv = fast::pow(r_cut_shifted, -n_pow);
+                Scalar rcutminv = fast::pow(r_cut_shifted, -m_pow);
                 pair_eng -= repulsive * rcutninv - attractive * rcutminv;
```

Comment:
```suggestion
                Scalar r_cut = fast::sqrt(rcutsq);
                Scalar r_cut_shifted = r_cut - delta;

                Scalar r_cut_n_inv = fast::pow(r_cut_shifted, -n_pow);
                Scalar r_cut_m_inv = fast::pow(r_cut_shifted, -m_pow);
                pair_eng -= repulsive * r_cut_n_inv - attractive * r_cut_m_inv;
```
Consistent and more underlining.